packages/noports_core/lib/src/common/types.dart

```diff
@@ -1,12 +1,21 @@
 import 'package:noports_core/sshrv.dart';
 
-typedef SshrvGenerator<T> = Sshrv<T> Function(String, int, {int localSshdPort});
+typedef SshrvGenerator<T> = Sshrv<T> Function(
+  String,
+  int, {
+  required int localPort,
```

Comment:
renamed localSshdPort to localPort in anticipation of future daemon versions which connect to ports other than the sshd port.

Also made it `required` to make sure all callers are being explicit